corgi/tasks/errata_tool.py

```diff
@@ -112,18 +127,23 @@ def slow_load_errata(erratum_name, force_process: bool = False):
     logger.info("Finished processing %s", erratum_id)
 
 
-def _get_relation_build_ids(erratum_id: int) -> set[int]:
+def _get_relation_builds(erratum_id: int) -> list[tuple[int, str]]:
```

Comment:
yup, I need to set it to this though, I guess because the tuple could be any length?
`_get_relation_builds(erratum_id: int) -> tuple[tuple[int, str],...]`